pkgs/development/web/bloomrpc/default.nix

```
@@ -0,0 +1,92 @@
+{ lib, stdenv, makeWrapper, fetchurl, dpkg, alsaLib, atk, cairo, cups, dbus, expat
+, fontconfig, freetype, gdk-pixbuf, glib, gnome2, pango, nspr, nss, gtk3, gtk2
+, at-spi2-atk, gsettings-desktop-schemas, gobject-introspection, wrapGAppsHook
+, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
+, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, nghttp2
+, libudev0-shim, glibc, curl, openssl, autoPatchelfHook }:
+
+let
+  runtimeLibs = lib.makeLibraryPath [
+    curl
+    glibc
+    libudev0-shim
+    nghttp2
+    openssl
+    stdenv.cc.cc
+  ];
+in stdenv.mkDerivation rec {
+  pname = "bloomrpc";
+  version = "1.5.2";
+
+  src = fetchurl {
+    url =
+      "https://github.com/uw-labs/bloomrpc/releases/download/${version}/bloomrpc_${version}_amd64.deb";
```

```suggestion
    url = "https://github.com/uw-labs/bloomrpc/releases/download/${version}/bloomrpc_${version}_amd64.deb";
```

Can we build from source instead?